client/policy_validator.go

```diff
@@ -25,13 +24,12 @@ import (
 
 type PolicyValidationResult struct {
 	err             error
+	crdDefined      bool
```

Comment:
Maybe 'enabled' would be a better name for this field?

Reply:
Certainly more friendly. I will change it.